docs/user-guide/options.md

```diff
@@ -212,6 +212,12 @@ CLI flags: `--report-needless-disables, --rd`
 
 Report `stylelint-disable` comments that don't actually match any lints that need to be disabled.
 
+## `validate`
+
+CLI flags: [`--validate, --no-validate`](cli.md#--validate---no-validate)
+
+Force enable/disable the validation of the rules' options.
```

Comment:
[suggestion] It seems helpful to have a description for the default value, e.g.

```suggestion
Force enable/disable the validation of the rules' options. Default: `true`.
```

Reply:
I would have advised to do the same for `formatter` but it's being changed in #7826.